src/Compilers/CSharp/Test/Syntax/Parsing/LambdaParameterParsingTests.cs

```diff
@@ -3035,5 +3035,445 @@ public void AsyncAwaitInLambda()
             }
             EOF();
         }
+
+        [WorkItem(60661, "https://github.com/dotnet/roslyn/issues/60661")]
+        [InlineData(LanguageVersion.CSharp9)]
+        [InlineData(LanguageVersionFacts.CSharpNext)]
+        [Theory]
+        public void KeywordParameterName_01(LanguageVersion languageVersion)
+        {
+            string source = "int =>";
```

Comment:
Is this only an issue for non-parenthesized lambdas? What if we had `(int, int) =>`?

Reply:
It might also be good to just test `(int) =>` as well.